index.html

```diff
@@ -40,31 +39,185 @@ <h1 class="heading_second">
+        <div class="side_info">
+          <h3 class="project_name">
+            Multi-Post<br>
+            Stories
+          </h3>
+          <ul class="first_list">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="tech_list">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="btn1" type="button">See Project</button>
+        </div>
+      </div>
+    </div>
+    <section id="secondlast_section">
+      <div class="section-last-conatiner">
+        <div class="home_containe">
+          <div>
+            <h1 class="aboutdiv">About<br>Myself</h1>
+          </div>
+          <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my
+            work and experience! If you like what you see and have a project you need coded, don't hestiate to contact
+            me.
+          </p>
+          <span>LET'S CONNECT</span>
+          <ul class="social_media">
+            <li><a href="https://twitter.com/silviatofana"><img src="./images/twitter.svg" alt="twitter"></a></li>
+            <li><a href="https://www.linkedin.com/in/silviatofana"><img src="./images/Linkedin_icon.svg"
+                  alt="linkedin"></a>
+            </li>
+            <li><a href="#"><img src="./images/images4.svg" alt="ok"></a></li>
+            <li><a href="#"><img src="./images/images5.svg" alt="github"></a></li>
+            <li><a href="#"><img src="./images/images6.svg" alt="medium"></a></li>
+          </ul>
+          <button class="btn2" type="button">Get My Resume</button>
+        </div>
+        <div class="languages-container">
+          <div>
+            <div class="languages_logo">
+              <h4 class="languages_heading">Languages</h4>
+              <div class="image_float">
+                <img src="./images/IconChevronDownLarger.png" alt="icon">
+              </div>
+            </div>
+            <ul class="languages languages-J">
+              <li class=“ellipse1”>
+                <img src="./images/Ellipse_j.png" alt="icon">
+                <span>JavaScript</span>
+              </li>
+              <li class="ellipse1">
+                <img src="/images/Ellipse_h.png" alt="icon">
+                <span>CSS</span>
+              </li>
+              <li class="ellipse1">
+                <img src="./images/Ellipse_.png" alt="con">
+                <span>HTML</span>
+              </li>
+            </ul>
+          </div>
+          <div class="languages_frameworks">
+            <h4>Frameworks</h4>
+            <span>&#62;</span>
+          </div>
+          <hr>
+          <div class="languages_skills">
+            <h4>Skills</h4>
+            <span id="span">&#62;</span>
+          </div>
+          <hr>
+        </div>
```

Comment:
- [ ] Please make sure to follow the guidelines of your figma template, here you're missing some points like: font-family, spacing, chevron icon (that arrow ">"). Please fix this issue

![image](https://user-images.githubusercontent.com/52635378/164917215-84788928-6c6b-434a-a5a5-6d9da55b9739.png)